model/llm/prompt/parse.go

```diff
@@ -21,7 +20,7 @@ func ParseResponse(response string) (assessment metrics.Assessments, code string
 
 	// Check for empty responses.
 	if strings.TrimSpace(response) == "" {
-		return assessment, "", pkgerrors.New("empty response from model")
```

Comment:
- [ ] Is the empty reponse now in the logs? It should. Then we should test that it is in the logs.
- [ ] Is it available in the CSV? It should. We should test for it.

Reply:
Added tests.